python/packaging.bzl

```diff
@@ -158,14 +162,28 @@ def py_wheel(name, twine = None, publish_args = [], **kwargs):
 
     _py_wheel(name = name, **kwargs)
 
-    if twine:
-        if not twine.endswith(":pkg"):
-            fail("twine label should look like @my_twine_repo//:pkg")
-        twine_main = twine.replace(":pkg", ":rules_python_wheel_entry_point_twine.py")
+    twine_args = []
+    if twine or BZLMOD_ENABLED:
         twine_args = ["upload"]
         twine_args.extend(publish_args)
         twine_args.append("$(rootpath :{})/*".format(_dist_target))
 
+    if BZLMOD_ENABLED or str(Label(twine)) == str(Label("//tools/publish:twine")):
```

Comment:
Maybe it should be a separate arg? `twine_binary` instead of `twine`. This would allow passing an arbitrary binary, so long as it supported the CLI args.

I guess in retrospect, `twine_lib` would have been a better name than `twine`. Ah well.